### PR TITLE
importccl: fix ALTER TABLE ... SET DEFAULT / SET NOT NULL

### PR DESCRIPTION
**Pull request by otan**
Based on #53808, only last commit is relevant.

----

Regression introduced in 921c60a0af19226b57274c070cdb5ca5d14a54d7.

Release justification: low risk, high benefit changes to
existing functionality

Release note: None

### CODE REVIEW

File: pkg/ccl/importccl/read_import_pgdump.go [hunk 1]
```diff
@@ -383,8 +383,10 @@ func readPostgresStmt(
 			case *tree.AlterTableSetDefault:
 				for i, def := range create.Defs {
 					def, ok := def.(*tree.ColumnTableDef)
+					// If it's not a column definition, or the column name doesn't match,
+					// we're not interested in this column.
 					if !ok || def.Name != cmd.Column {
-						return errors.Errorf("unsupported %T definition: %s", def, def)
+						continue
```
Comment (dt) [inline pkg/ccl/importccl/read_import_pgdump.go:389]: would it make sense to have a `found` bool and error out if we never find a def with name == cmd.Column? or just assume it won't happen?

**Review by dt — APPROVED** [review]
good catch! Thanks!

File: pkg/ccl/importccl/read_import_pgdump.go (hunk 1 above)
Comment (otan) [reply pkg/ccl/importccl/read_import_pgdump.go:391]: yeah, done!

**Comment by otan** [comment]
bors r=dt

**Pull request merged**